client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/form-content/notification-duration.tsx

```diff
@@ -54,9 +57,11 @@ export default function NotificationDuration( {
 						key={ duration.time }
 						selected={ duration.time === selectedDuration?.time }
 						onClick={ () => selectDuration( duration ) }
-						disabled={ duration.paid_tier && ! enablePaidDurations }
+						disabled={ restriction !== 'none' && duration.paid_tier }
 					>
-						{ duration.label }
+						{ duration.label }{ ' ' }
```

Comment:
Sorry I'm late to the review here: rather than an explicit non-breaking space, it would be more in line with Calypso standards to use CSS to manage spacing and wrapping.